builder/bundles.go

```diff
@@ -1122,7 +1122,7 @@ func createVersionsFile(baseDir string, packagerCmd []string) error {
 	return w.Flush()
 }
 
-func updateOSReleaseFile(b *Builder, filename, version string, homeURL string) error {
+func updateOSReleaseFile(b *Builder, filename, version string, homeURL string, upstreamVer string) error {
 	fmt.Println("... updating os-release file")
 	//Replace the default os-release file if customized os-release file found
```

Comment:
Not related to this PR, but can you move this print statement to before line 1148 `var newBuf bytes.Buffer`
Right now both the update and create message for os-release is being displayed when creating.